gordon_gcp/schemas/audit-log.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "id": "https://github.com/spotify/gordon-gcp/blob/master/gordon_gcp/schemas/audit-log.schema.json",
+    "title": "Google Audit Log Messages",
+    "description": "Schema for parsing Google Audit Log Messages",
+    "required": ["protoPayload", "logName"],
+    "properties": {
+        "protoPayload": {
+            "type": "object",
+            "properties": {
+                "methodName": {
+                    "type": "string",
+                    "description": "Action performed on a GCP",
+                    "enum": [
+                        "v1.compute.instances.insert",
+                        "v1.compute.instances.delete"
+                    ]
+                },
+                "resourceName": {
+                    "type": "string",
+                    "description": "",
+                    "examples": [
+                        "projects/a-project-id/zones/a-zone-name/instances/an-instance-name"
+                    ],
+                    "pattern": "^projects/[0-9]+/zones/[a-z\\-0-9]+/instances/[a-z\\-0-9]+"
+                }
+            },
+            "required": ["methodName", "resourceName"]
+        },
+        "logName": {
+            "type": "string",
+            "description": "Name of audit log",
+            "examples": [
+                "projects/a-project-id/logs/cloudaudit.googleapis.com%2Factivity"
+            ],
+            "pattern": "^projects/[a-z][a-z\\-0-9]{5,29}/logs/cloudaudit.googleapis.com%2Factivity"
+        },
+        "receiveTimestamp": {
+            "type": "string",
+            "description": "Time the log entry was received by Stackdriver Logging",
+            "examples": [
+                "2018-01-01T23:13:45.123456789Z"
+            ],
+            "pattern": "^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(\\.[0-9]+)?(([Zz])|([\\+|\\-]([01][0-9]|2[0-3]):[0-5][0-9]))"
```

Comment:
I suck at regexes - is there a better one for RFC 3339 format? Per [log entry operation docs](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogEntryOperation).

Reply:
The [docs](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#top_of_page) (I linked to a wrong anchor id earlier - sorry!) to refer to the [Timestamp](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp) type that conforms to [RFC3339](https://www.ietf.org/rfc/rfc3339.txt), which does allow both `T`/`Z` and `t`/`z`.

The `\d+` does indeed work \o/ and the other suggested cleanups seem to work just fine too :D @descent3me I'm coming to you for all my regex needs from now on 😉